### ISSUE TITLE
No voice on list

### ISSUE THREAD

**Issue by kowaldemar641** [j-kaltes/Juggluco#61]
A install juggluco and i have't voice to speak glucose value

**Comment by j-kaltes** [comment]
Set Language of Juggluco to English and read Left middle menu->Talk->Help.

**Comment by kowaldemar641** [comment]
It's not working. I have not voice to select

**Comment by j-kaltes** [comment]
Did you read the following in Left middle menu->Talk->Help?
_On some phones the Android settings for text-to-speech have to be changed for it to work. Sometimes the data of a language has to be installed or the configuration changed. On Samsung phones "Google text-to-speech" has to be set instead of "Samsung text-to-speech engine" in order to choose from different voices. These options can be found by searching for text-to-speech in Android settings or going to Language and input (->Advanced)->Text-to-Speech. Here you can select "Speech Services by Google" and install voice data. In other phones it is under System settings->Accessibility->Vision->Text-to-Speech settings._

**Comment by kowaldemar641** [comment]
Yes, i have samsung phone. When i set speech Services by Google juggluko speak polish... THANKS

**Comment by bobnil** [comment]
I can confirm if worked when changed the voice engine from Samsung to Google.

1. Go to your phone setting, search for text-to-speech
2. If the preferred engine is Samsung, press the "Preferred engine" header, not the settings symbol.
3. Change to Googles engine
4. Restart your phone or force stop Juggluco for it to recognize the change.

Now the voice list should be populated with available voices.
